2022/microsoftUCuenca/data.json

```diff
@@ -0,0 +1,27 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
```

Comment:
We would like to know who your valid folder maintainers are, please!

Reply:
I'm going just to ad @ronnytn to the valid maintainers and accept the PR. There is no Sessionize API Id so those sessions doesn't appear on the agenda, but that's OK. 

PS: @ronnytn if you still desire your sessions to appear in the global agenda, please follow the instructions available at https://blog.globalazure.net/Blog/Post/284/Sessionize-integration-setup-for-local-event-organizers-for-2022-edition and do another PR adding the Sessionize API Id to the `data.json` file